sklearn/manifold/_t_sne.py

```diff
@@ -597,6 +598,16 @@ class TSNE(BaseEstimator):
 
         .. versionadded:: 0.22
 
+    square_distance : {True, 'legacy'}, default='legacy'
+        Whether distance values are squared before t-SNE is run. ``'legacy'``
+        means that distances are squared only when ``metric="euclidean"``.
+        In version 0.26, ``'legacy'`` will be removed as a setting, and the
```

Comment:
That should go in a `.. deprecated::` section, with a message like "`square_distance='legacy'` is deprecated in 0.24 and will not be possible in 0.26. The default value for this parameter will change to `True`".

Reply:
Ah, of course, thank you! :)